thetanuts/thetanuts_test.py

```diff
@@ -181,20 +183,13 @@ def get_env(variable: str) -> str:
     )["errors"]
     is False
 ):
-    print("Bid validated")
+    print(" -> Bid validated")
```

Comment:
@mdantonio maybe we should propose a future PR to replace all "print" in the SDK with the python logger ;=) 

but this is fine for now